src/components/v2/shared/SplitItem.tsx

```diff
@@ -81,24 +76,21 @@ export default function SplitItem({
       }
       return t`Unknown Project`
     }
+
     return (
       <div>
         <div>
-          <Tooltip title={getProjectTooltip()}>
-            <span>
-              <Link
-                href={v2ProjectRoute({ projectId: split.projectId })}
-                target="_blank"
-              >
-                <a
-                  className="text-primary hover-text-action-primary hover-text-decoration-underline"
-                  style={{ fontWeight: 500 }}
-                >
-                  {getProjectLabel()}
-                </a>
-              </Link>
-            </span>
-          </Tooltip>
+          <Link
+            href={v2ProjectRoute({ projectId: split.projectId, handle })}
+            target="_blank"
```

Comment:
The redirect isn't working properly - I think maybe this needs to be on the a tag itself

Reply:
Ahh great great catch.

Fixed